tts/src/main/java/com/caneseeproject/tts/tts.kt

```diff
@@ -0,0 +1,45 @@
+package com.caneseeproject.tts
+
+import android.speech.tts.TextToSpeech
+import android.util.Log
+import java.util.*
+import com.caneseeproject.sensorPortals.SensorReading
+
+interface TexrToSpeechInterface {
+    fun onInit(status: Int)
+    fun speakOut(text: String)
+}
+
+class TextToSpeachClass : TexrToSpeechInterface {
+    var tts: TextToSpeech? = null
+    override fun onInit(status: Int) {
+        if (status == TextToSpeech.SUCCESS) {
+            // set US English as language for tts
+            val result = tts!!.setLanguage(Locale.US)
+
+            if (result == TextToSpeech.LANG_MISSING_DATA || result == TextToSpeech.LANG_NOT_SUPPORTED) {
+                Log.e("TTS","The Language specified is not supported!")
+            }
+        } else {
+            Log.e("TTS", "Initilization Failed!")
+        }
+    }
+    override fun speakOut(text: String) {
+        //tts function implementation
+        val textString = text.toString()
+        tts!!.speak(textString, TextToSpeech.QUEUE_FLUSH, null);
+    }
+
+    fun notify(RecentNotification: NotificationType) =
```

Comment:
In Kotlin, parameters should follow the `lowerCamelCase` rule: `recentNotification`.